.github/workflows/docker.yml

```diff
@@ -28,5 +33,24 @@ jobs:
       - name: Publish container
         working-directory: docker
         run: |
-          docker login -u "$DOCKERHUB_USERNAME" -p "$DOCKERHUB_PASSWORD"
           make publish
+      - name: Build release container
```

Comment:
Will these get triggered the same way as the dev/non-release builds?

Reply:
yep, this also runs on every commit, but just realized this container is lacking the logic to only build a new release container if one hasn't already been published for the latest version

Reply:
okay cool, yeah, I was wondering what was preventing it from building release containers for dev versions